config/runtime.exs

```diff
@@ -36,7 +36,9 @@ if config_env() != :test do
     message_log_s3_folder: System.get_env("MESSAGE_LOG_S3_FOLDER"),
     message_log_report_s3_folder: System.get_env("MESSAGE_LOG_REPORT_S3_FOLDER"),
     monitoring_api_key: System.get_env("MONITORING_API_KEY"),
-    active_headend_path: System.get_env("ACTIVE_HEADEND_S3_PATH")
+    active_headend_path: System.get_env("ACTIVE_HEADEND_S3_PATH"),
+    active_pa_messages_url: System.get_env("ACTIVE_PA_MESSAGES_URL"),
```

Comment:
On second thought @panentheos , I'm not sure if I interpreted your comment about the runtime env here correctly. Did you mean that it's not necessary to split the screenplay base url and the route for active PA messages or that we don't need the screenplay url as an env variable at all? I assume we still want it as an env variable for local testing and such, right?

Reply:
My thinking was: The screenplay base URL should be an env variable, like we do for other systems. The path for the endpoint should not be: Ideally, it wouldn't be abstracted at all, since the actual API path on Screenplay is always the same. However, because of how the `Fake` module works, we need some sort of hook for testing different responses. The least intrusive thing that gets us that would be to put the path in the _application_ environment, with the real value configured statically in `config.exs`, and then have the tests modify the application env like they're doing.